addons/account_edi_ubl_cii/models/account_edi_xml_cii_facturx.py

```diff
@@ -214,6 +214,18 @@ def grouping_key_generator(base_line, tax_values):
             sum_fixed_taxes = sum(x['amount'] for x in line_vals['allowance_charge_vals_list'])
             line_vals['line_total_amount'] = line_vals['line'].price_subtotal + sum_fixed_taxes
 
+            line_vals['quantity'] = line_vals['line'].quantity #/!\ The quantity is the line.quantity since we keep the unece_uom_code!
+
+            # Invert the quantity and the price if a line has a negative price total
+            if line_vals['line'].currency_id.compare_amounts(line_vals['gross_price_total_unit'], 0) == -1:
+                line_vals['quantity'] *= -1
+                line_vals['gross_price_total_unit'] *= -1
+                line_vals['price_unit_after_discount'] *= -1
```

Comment:
gross_price_total_unit, price_unit_after_discount: are not used in the `cii_22_templates.xml` or are they ?